README.md

```diff
@@ -0,0 +1,48 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Bookstore
+
```

Comment:
- [ ] Please it will be nice to include a short description about your project as this will enable users to quickly get an idea of what your project is about.